_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -0,0 +1,306 @@
+
+There are multiple scenarios for installation depending on what GCP components (GKE, Forseti) are installed, if any.  After addressing the [pre-requisites](#install-pre-requisites), please use the following table to determine from which step you should start. 
+
+{: .table}
+|  An Empty GCP Project  |  Forseti Infrastructure  |  GKE Cluster  |  Then start at  |
+|-------|-------|-------|-------|
+| Yes | No | No | [Deploy Forseti and Forseti on GKE end-to-end](#deploy-terraform-and-forseti-on-gke-end-to-end) |
+| No | Yes | No | [Deploy Forseti in a New GKE Cluster](#deploy-forseti-in-a-new-gke-cluster) |
+| No | Yes | Yes | [Deploy Forseti in an Existing GKE Cluster](#deploy-forseti-in-an-existing-gke-cluster) |
+| No | Yes | Yes | [Deploy Forseti in an Existing GKE Cluster (Helm)](#deploy-forseti-in-an-existing-gke-cluster-helm) |
+
+## Install Pre-Requistes
+
+The following tools are required:
+* [Terraform](https://www.terraform.io/downloads.html) - 0.12.x
+* [gsutil](https://cloud.google.com/storage/docs/gsutil)
+
+## Deploying with Terraform
+
+In each of the following scenarios, the user will create a *main.tf* file and add the appropriate input variables.  It is recommended that this *main.tf* be created in an empty directory.
+
+Each scenario described below invokes a corresponding example in the [examples/on_gke](https://github.com/forseti-security/terraform-google-forseti/tree/master/examples/on_gke) folder of the *terraform-google-forseti* Terraform module.  Please understand that each of these examples are just that, examples.  Each example has a *main.tf* file that describes how the environment will be built addressing common scenarios.  Please review the examples to determine if the examples are sufficient for the environment where Forseti is deployed.
+
+Wherever possible, the examples utilize [modules](https://registry.terraform.io/modules/terraform-google-modules) developed and curated by the [Cloud Foundation Toolkit](https://cloud.google.com/foundation-toolkit/) team.  These modules implement opinionated best practices for deploying GCP components.  For example, the [kubernetes-engine](https://registry.terraform.io/modules/terraform-google-modules/kubernetes-engine/google/4.0.0) module applies practices found in the [GKE hardening](documentation).
+
+### Deploy Forseti and Forseti on GKE end-to-end
+
+Create a file named *main.tf* in an empty directory and add the following content.  Add the appropiate values for each of the input vairables (e.g. domain, gsuite_admin_email).
+
+```bash
+module "forseti-on-gke-end-to-end" {
+    source                  = "terraform-google-modules/forseti/google//examples/on_gke_end_to_end"
```

Comment:
Remove extra slash: `/google//examples`

Reply:
Actually that's a Terraform thing.  When I remove the extra slash, Terraform complains it can't find that folder.

Reply:
Ah ok - closing this comment. Thanks!